flocker/acceptance/test_environment_variables.py

```diff
@@ -247,12 +245,15 @@ def get_mysql_node_2(ignored):
             Move MySQL to ``node_2`` and return a ``Deferred`` which fires
             with a connection to the previously created database on ``node_2``.
             """
+            # Listen on different port so it's clear we're connecting to
```

Comment:
Is this absolutely necessary? Could it not be shown just as well with `assert_expected_deployment`? In fact isn't that the point of having `test_moving_mysql` which now seems redundant?